scripts/http-vuln-cve2015-6396.nse

```diff
@@ -0,0 +1,126 @@
+local shortport = require "shortport"
+local http = require "http"
+local stdnse = require "stdnse"
+local string = require "string"
+local vulns = require "vulns"
+
+description = [[
+Executes remote code by exploiting the CVE-2015-6396 vulnerability in the following Cisco devices:
+
+- RV110W
+- RV130W
+- RV215W
+
+The CLI command parser on the above Cisco devices allows local users to execute arbitrary shell commands
+as an administrator via crafted parameters. Authentication is required. You can possibly get the device
+credentials by exploiting the CVE-2014-0683 vulnerability on the device.
+
+References:
+* https://nvd.nist.gov/vuln/detail/CVE-2015-6396
+]]
+
+---
+-- @usage nmap -sV --script http-vuln-cve2015-6396 --script-args user='cisco' pwd='cisco' cmd='reboot' <target>
+-- @output
+-- PORT   STATE SERVICE REASON
+-- 443/tcp open  https   syn-ack
+-- | http-vuln-cve2015-6396:
+-- |   VULNERABLE:
+-- |   Remote Code Execution in Cisco V110W, RV130W and RV215W (CVE-2015-6396)
+-- |     State: VULNERABLE (Exploitable)
+-- |     IDs:  CVE:CVE-2015-6396
+-- |       A remote code execution vulnerability exists on the CLI command parser which allows local users
+-- |       to execute arbitrary shell commands as an administrator via crafted parameters. Authentication
+-- |       is required. You can possibly get the device credentials by exploiting the CVE-2014-0683
```

Comment:
I don't see the point of this message as it will only be shown if the users already provided the credentials.